test/collection/test_config.py

```diff
@@ -82,6 +82,22 @@ def test_basic_config():
             }
         },
     ),
+    (
+        Configure.Vectorizer.multi2vec_cohere(
+            model="embed-multilingual-v2.0",
+            truncate="NONE",
+            vectorize_collection_name=False,
+            base_url="https://api.cohere.ai",
+        ),
+        {
+            "multi2vec-cohere": {
+                "model": "embed-multilingual-v2.0",
```

Comment:
Propose `embed-multilingual-light-v3.0` for the same reason. I guess the test must be using text input only?

Reply:
yep, this only tests that the config is created correctly, there is no name validation